docs/future_improvements.md

```diff
@@ -1,3 +1,76 @@
+# Open Auctions
+
+The most widely valuable use case for auctions is the Open Auction, which allows anyone to participate in the auction as bidders.
+
+## Problem Statement (Persistent Deposit Sufficiency)
+
+The chief difficulty with running open auctions is ensuring the bidder deposit is sufficient and remains until the bidder withdraws from the auction.
+
+In private auctions we can easily bypass this problem by offloading the responsibility to the seller, who is both incentivized to ensure the deposit is sufficient, and a trusted party in regards to the auction.
+
+Public auctions must solve this same problem without the involvement of the seller after the initial announcing of the auction. In order to do this, we will rely on the other class of trusted parties in the auction protocol - the hydra delegates.
+
+## Solution
+
+### L1
```

Comment:
Shall we mention that we no longer need seller authorization signatures for open auctions @SamDelaney?